extensions/amp-consent/0.1/consent-policy-manager.js

```diff
@@ -28,6 +28,12 @@ export const MULTI_CONSENT_EXPERIMENT = 'multi-consent';
 const CONSENT_STATE_MANAGER = 'consentStateManager';
 const TAG = 'consent-policy-manager';
 
+const UNBLOCK_ON_ALL_POLICY = 'consent-unblock-on-all';
```

Comment:
`data-block-on-consent=consent-unblock-on-all` reads weird.

how about `data-block-on-consent=none` and `data-block-on-consent=all`. The latter is the current default.

Reply:
good point. As discussed offline I renamed it to `_none`. Unfortunately I can't change `default` now because we documented on how to overwrite `default` policy, if you think it's worth changing it, we can only map `default` to `_all` somewhere in our codebase.